packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -35,11 +35,15 @@ suite('requests helpers', ({ expect, stub, spy }) => {
     const query = 'dress';
     const collection = 'alternate';
     const refinements = [{ a: 'b' }, { c: 'd' }];
+    let state;
+    let sessionId;
     let sortSelector: sinon.SinonStub;
     let requestSortAdapter: sinon.SinonStub;
     let pastPurchaseBiasingAdapter: sinon.SinonStub;
 
     beforeEach(() => {
+      state = <any>{ session: { sessionId: 'foo' } };
+      sessionId = <any>RequestHelpers.extractSessionId(state);
```

Comment:
I don't think we want to derive `sessionId` *from* `extractSessionid()`. Rather, we should be doing defining `sessionId` as some primitive value, then defining `state` using the resulting variable.

Reply:
Right, I will make the changes. Resolving conversation.